### DISCUSSION TITLE
How do I add a custom week to Kade Engine 1.4.1?

### DISCUSSION THREAD

**Discussion by MaxyRay** [KadeArchive/Kade-Engine#189] [Q&A] [answered]
How I do it ._.

**Comment by theDetourist** [comment, marked as answer]
First, [watch this](https://www.youtube.com/watch?v=ac94p0zqqYM).
Then, tell us how much progress you've managed to make based on that.

And if that's not enough, I'm sure a quick Youtube or Google search shall give you fine results.

**Reply by MaxyRay** [reply]
> First, [watch this](https://www.youtube.com/watch?v=ac94p0zqqYM).
> Then, tell us how much progress you've managed to make based on that.
> 
> And if that's not enough, I'm sure a quick Youtube or Google search shall give you fine results.

Thank you very much, I appreciate it, I was referring to the kade engine 1.4.1 since in that version the codes change but the video served me, really thank you very much

**Reply by isakube** [reply]
https://github.com/KadeDev/Kade-Engine/discussions/226
